lib/lotus/view/configuration.rb

```diff
@@ -388,6 +410,7 @@ def reset!
         root             DEFAULT_ROOT
         default_encoding DEFAULT_ENCODING
 
+        @partials   = ::Hash.new { |h, k| h[k] = Hash.new }
```

Comment:
while i like the simplified syntax on the lookup side, i think that @stevehook is right when he mentions "lookup mutating the state" as a a downside.